cmd/src/actions_scope_query.go

```diff
@@ -52,8 +53,19 @@ Examples:
 			return err
 		}
 
+		// Convert action file to JSON, if it was yaml.
+		jsonActionFile, err := yaml.YAMLToJSONStrict(actionFile)
```

Comment:
Same here: we need to be backwards compatible and still allow reading JSON files.